phoenicis-tools/src/main/java/org/phoenicis/tools/lnk/LnkFileAttributeFlagsParser.java

```diff
@@ -0,0 +1,30 @@
+package org.phoenicis.tools.lnk;
+
+/**
+ * The FileAttributesFlags structure defines bits that specify the file attributes of the link target, if the target is
+ * a file system item. File attributes can be used if the link target is not available, or if accessing the target would
+ * be inefficient. It is possible for the target items attributes to be out of sync with this value.
```

Comment:
What does `out of sync` mean?

Reply:
Copied and paste from the official documentation